PlaceRouteHierFlow/placer/design.cpp

```diff
@@ -280,6 +280,20 @@ design::design(PnRDB::hierNode& node, PnRDB::Drc_info& drcInfo, const int seed)
     this->Port_Location.back().tid = it->tid;
     this->Port_Location.back().pos = placerDB::Bmark(it->pos);
   }
+  // Add spread constraints
```

Comment:
It can save some memory only when *itb1< *itb2

Reply:
Addressed now.